dotcom-rendering/src/web/components/CommercialMetrics.tsx

```diff
@@ -16,7 +17,16 @@ export const CommercialMetrics: React.FC<{
 		false,
 	);
 
+	const [adBlockerInUse, setAdBlockerInUse] = useState<boolean>();
+
+	isAdBlockInUse().then((value) => {
```

Comment:
@mxdvl I think we have to do this here rather than use `useAdBlockInUse`. That function only sets the value of `isInUse` [here](https://github.com/guardian/dotcom-rendering/blob/150fc2d81e6a66d9c3336185e874fc8cd0288546/dotcom-rendering/src/web/lib/useAdBlockInUse.ts#L17) if `isAdBlockInUse` returns true, but won't set it to false.

Reply:
Is there ever a case where someone would turn off their ad blocker without refreshing the page and we’d want to re-render? I think `useAdBlockInUse` is fine.

Reply:
The case that `useAdBlockInUse` doesn't work for is when the user doesn't have an ad blocker. `isAdBlockInUse` returns `false`, `isInUse` is not updated, the callback in `useEffect` isn't executed (because `adBlockerInUse` remains `undefined`)

Reply:
Gotcha, makes sense.